extension/chrome/elements/compose-modules/compose-quote-module.ts

```diff
@@ -88,8 +88,7 @@ export class ComposeQuoteModule extends ViewModule<ComposeView> {
     try {
       const { raw } = await this.view.emailProvider.msgGet(msgId, 'raw', progress => this.setQuoteLoaderProgress(progress));
       this.setQuoteLoaderProgress('processing...');
-      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-      const decoded = await Mime.decode(Buf.fromBase64UrlStr(raw!));
+      const decoded = await Mime.decode(Buf.fromBase64UrlStr(raw || ''));
```

Comment:
calling `Mime.decode` on an empty string? To get an empty structure?
and then call `Mime.processDecoded` on the empty structure?
Wouldn't it be better to simply `return undefined` -- the function return type is `Promise<MessageToReplyOrForward | undefined>`?

Reply:
That is right. I have added a check the value of `raw` and `return undefined` if it is indeed `undefined`. thank you. I learned from this.